src/config_brain_memory.js

```diff
@@ -108,8 +106,24 @@ brain.prepareMemory = function() {
     }
   }
 
-  if (config.stats.summary) {
+  if (config.stats.enabled) {
+    Memory.stats = {};
+    var name = Memory.username;
+    if (name) {
+      Memory.stats[name + '.cpu.limit'] = Game.cpu.limit;
+      Memory.stats[name + '.cpu.tickLimit'] = Game.cpu.tickLimit;
+      Memory.stats[name + '.cpu.bucket'] = Game.cpu.bucket;
+      Memory.stats[name + '.exec.halt'] = Game.cpu.bucket < Game.cpu.tickLimit * 2;
 
+      Memory.stats[name + '.gcl.level'] = Game.gcl.level;
+      Memory.stats[name + '.gcl.progress'] = Game.gcl.progress;
+      Memory.stats[name + '.gcl.progressTotal'] = Game.gcl.progressTotal;
+
+      Memory.stats[name + '.rooms.available'] = Game.rooms.length;
+    }
+  }
```

Comment:
Yes, originally, the screeps stats programs didn't allow it, shortly after node-agent started using it it got merged back into the other projects